### ISSUE TITLE
Redefine var not showing error

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#3730]
Should this show an error? me_not_null is already defined but I can compile it and get a null on the second print.

```
func _ready():
    bug("me_not_null")

func bug(me_not_null):
    print(me_not_null)
    var me_not_null = me_not_null
    print(me_not_null)
```

**Comment by bojidar-bg** [comment]
Close?

**Comment by JarLowrey** [comment]
Just ran into this issue in 2.1.4 and had no idea what was going on for quite awhile. Why was it closed?

**Comment by akien-mga** [comment]
Because it was fixed by #4918 (for Godot 3.0).
